controls/SV-204421.rb

```diff
@@ -33,8 +33,7 @@
       next unless user(user).uid >= 1000
 
       describe shadow.users(user) do
-        its('max_days.first') { should cmp input('expected_max_password_lifetime') }
-        its('max_days.first') { should cmp <= input('max_password_lifetime') }
+        its('max_days.first') { should cmp <= input('password_lifetime') }
```

Comment:
It looks like based on the edit, this _should_ be `max_password_lifetime` given the comment above the input in the inspec.yml and this check is a lte?

Reply:
Good catch. I renamed them so there is one min and one max.